sentry/src/main/java/io/sentry/util/CheckInUtils.java

```diff
@@ -30,18 +30,19 @@ public static <U> U withCheckIn(
       final @Nullable MonitorConfig monitorConfig,
       final @NotNull Callable<U> callable)
       throws Exception {
-    final @NotNull IHub hub = Sentry.getCurrentHub();
+    final @NotNull IScopes scopes = Sentry.getCurrentScopes();
     final long startTime = System.currentTimeMillis();
     boolean didError = false;
 
-    hub.pushScope();
-    TracingUtils.startNewTrace(hub);
+    // TODO fork instead
```

Comment:
That's addressed in follow up PR right?

Reply:
Yes, https://github.com/getsentry/sentry-java/pull/3347